scripts/update-dockerhub-description.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+set -euxf -o pipefail
+
+if [ "${WORKFLOW}" == "ci-docker-build.yml" ]; then
+  binary="docker"
```

Comment:
there is no such binary / image in Jaeger named `jaegertracing/docker`. We publish 2-3 dozen different images, you can see them all here: https://hub.docker.com/u/jaegertracing